apache-maven/src/assembly/shared/mvnvalidate.ps1

```diff
@@ -0,0 +1,7 @@
+# check mvn home
+$MAVEN_HOME = (Get-Item $PSScriptRoot).Parent
+
+# check if maven command exists
+if (-not (Test-path "$MAVEN_HOME\bin\mvn.ps1")) {
+    Write-Error -Message "Maven command ($MAVEN_HOME\bin\mvn.ps1) cannot be found" -ErrorAction Stop
+}
```

Comment:
I wonder whether this is a relic from the past. In `mvnvalidate` this check isn't present and this file is part of `mvn`/`mvn.cmd`/`mvn.ps1`. How can this fail?`

Reply:
If `mvn.ps1` or `mvn.cmd` is removed or not accessible.  

But it is indeed a corner case, shall we remove it?

Reply:
But how can this happen at all? This is a chicken-and-egg issue. This block is part of the merged `mvn.ps1`. If the file is not present that check will never happen, no? This confuses me.